src/tools/token.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
const jwt = require("jsonwebtoken")
```